code/game/objects/items/devices/hacktool.dm

```diff
@@ -0,0 +1,100 @@
+/obj/item/device/multitool/hacktool
+	var/is_hacking = 0
+	var/max_known_targets
+
+	var/in_hack_mode = 0
+	var/list/known_targets
+	var/list/supported_types
+	var/datum/topic_state/default/must_hack/hack_state
+
+/obj/item/device/multitool/hacktool/New()
+	..()
+	known_targets = list()
+	max_known_targets = 5 + rand(1,3)

```

Comment:
I'd be concerned if the overhead of an addition operation in a New() proc had any effect in the short or long term.
The purpose the statement was written as it was, was to clarify that 5 was the intended base amount and the rand() is just a bonus.